seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/mongodb/MongodbSourceToConsoleIT.java

```diff
@@ -0,0 +1,116 @@
+import lombok.extern.slf4j.Slf4j;
+import org.bson.Document;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.lifecycle.Startables;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+@Slf4j
+public class MongodbSourceToConsoleIT extends FlinkContainer {
+
+    private static final String MONGODB_IMAGE = "mongo:latest";
+    private static final String MONGODB_CONTAINER_HOST = "flink_e2e_mongodb";
+    private static final String MONGODB_HOST = "localhost";
+    private static final int MONGODB_PORT = 27017;
+    private static final String MONGODB_DATABASE = "test_db";
+    private static final String MONGODB_COLLECTION = "test_table";
+    private static final String MONGODB_URL = String.format("mongodb://%s:%d/%s?retryWrites=true&writeConcern=majority", MONGODB_HOST, MONGODB_PORT, MONGODB_DATABASE);
+
+    private GenericContainer<?> mongodbContainer;
+
+    private MongoClient client;
+
+    @BeforeEach
+    public void startRedisContainer() {
+        mongodbContainer = new GenericContainer<>(MONGODB_IMAGE)
+            .withNetwork(NETWORK)
+            .withNetworkAliases(MONGODB_CONTAINER_HOST)
+            .withExposedPorts(MONGODB_PORT)
+            .waitingFor(new HttpWaitStrategy()
+                .forPort(MONGODB_PORT)
+                .forStatusCodeMatching(response -> response == HTTP_OK || response == HTTP_UNAUTHORIZED)
+                .withStartupTimeout(Duration.ofMinutes(2)))
+            .withLogConsumer(new Slf4jLogConsumer(log));
+        Startables.deepStart(Stream.of(mongodbContainer)).join();
+        log.info("Mongodb container started");
+        given().ignoreExceptions()
+            .await()
+            .atMost(180, TimeUnit.SECONDS)
+            .untilAsserted(this::initConnection);
+        this.generateTestData();
+    }
+
+    private void initConnection() {
+        client = MongoClients.create(MONGODB_URL);
+    }
+
+    private void generateTestData() {
+        MongoCollection<Document> mongoCollection = client
+            .getDatabase(MONGODB_DATABASE)
+            .getCollection(MONGODB_COLLECTION);
+        mongoCollection.deleteMany(new Document());
+
+        HashMap<String, Object> map = new HashMap<>();
+        map.put("id", 1);
+        map.put("key_aa", "value_aa");
+        map.put("key_bb", "value_bb");
+        Document doc = new Document(map);
+        mongoCollection.insertOne(doc);
+    }
+
+    @Test
+    public void testRedisSource() throws IOException, InterruptedException {
```

Comment:
Please change to `testMongoSource`

Reply:
@TyrantLucifer Thank you for your reminder